Parsers/ASimProcessEvent/ProductParsers/ProcessEventMicrosoftEventCreate.yaml

```diff
@@ -0,0 +1,76 @@
+Parser:
```

Comment:
We have naming inconsistency. Authentication is WindowsEvents and this is MicrosoftEvents. I think it should be MicrosoftWindowsEvent is the correct one.